### ISSUE TITLE
in link select mode, control+enter needs to open the link in a background tab

### ISSUE THREAD

**Issue by bennnnn** [peterdotjs/quick-find#13]
actually I'll see if I can't add this one myself, a small addition like this would be pretty simple. In fact, I would prefer it if you let me, because I have zero github submission history.

**Comment by bennnnn** [comment]
before this extension, I was using ctrl+f, type, press f3 until link is selected, press escape, press ctrl+enter to load link in background tab. That is more functionality than what I have now, which restricts me to the current tab.

Please interrupt me if this is an impossible modification, it will save me much time context loading time.

**Comment by peterdotjs** [comment]
@bennnnn a pull request for this feature is more than welcome. And yes this should be doable.

**Comment by lastorset** [comment]
While you're (or someone™ is) at it, I'd also suggest adding other modifiers, like Ctrl+Shift to open in a foreground tab and Shift to open in a new window.

**Comment by peterdotjs** [comment]
@lastorset opening in a new tab versus new window is controlled by browser settings. Unless you are referring to creating a popup window which I would be against.

**Comment by lastorset** [comment]
My reference is the old behavior in Opera 12, which had a built-in feature with some of the functionality of this extension. One of the great things about that software was the consistency of keyboard shortcuts throughout the interface, so that Ctrl+click on link, Ctrl+back button, Ctrl+context menu back, Ctrl-Enter when searching for links etc. all caused the target to be opened in a background tab instead of the same tab. Same for Ctrl+Shift. (My memory of the exact modifier keys is fuzzy, but you get the idea.) I only realized I was missing this when I searched and wanted something opened in a foreground window.

In other words, what I was suggesting was just consistency with the rest of the browser interface, not advocacy of opening new windows. I rarely do that, so I'm not actually too fussed about the Shift modifier.

However, what do you have against popup windows when the user specifically requests them by holding down Shift? I wouldn't mind having the convenience those few times I actually need it.

(I couldn't find any option in Chrome or Opera to open windows instead of tabs. Not that that's an option I ever would enable!)
